### ISSUE TITLE
Actions and cohorts don't populate from events (custom or page view)

### ISSUE THREAD

**Issue by daichety** [PostHog/posthog#5062]
## Bug description

I usually use custom events and page view events to build actions and cohorts. Unfortunately since recently those don't populate although I can see the events coming in.
If I go via "Insights" and select the page view setting I want to use for my action (current url contains "success") I can see the sessions, but those just get transferred over, in order to be used as actions

## Expected behavior
As it was until beginning of June - when I create an action, I should be able see sessions accordingly

## How to reproduce

1. find for a valid event that you wan't to create an action from
2. go to actions and select it
3. see if it populates

## Environment

- [ ] PostHog Cloud
- [x] self-hosted PostHog, version/commit: not sure

**Comment by macobo** [comment]
Hey @daichety, sorry for taking long to get back to you.

Can you take a look at the instance status page and tell us what that's reporting? It's behind this green checkmark.
![image](https://user-images.githubusercontent.com/148820/128004976-2d128848-ea77-4502-9901-43f5686a528a.png)

**Comment by Twixes** [comment]
Hello @daichety, could you please provide us with your instance status as @macobo suggested? It's very difficult to diagnose the issue without that!
So the specific problem seems to be that actions aren't calculated as they come in. This is the only thing not working correctly and insights not being correct is caused by that, as I understand?

**Comment by daichety** [comment]
hi @macobo @Twixes,
sorry for the late response! 
Please find here the status page report. 
<img width="803" alt="Screenshot 2021-08-25 at 19 50 21" src="https://user-images.githubusercontent.com/63511071/130840549-35bae753-d3f1-44e1-a832-878339ab9b42.png">

Following the latest update to version 1.27.0 I can see events populating the actions.
It is however not picking up on actions retrospectively and I just see only very recent events associated with the action, although there are relevant events from the days before as well.
Just to clarify I am referring to a custom event. 

Furthermore, generating cohorts from the actions is not working either and it constantly shows "calculating" only.

Thanks for looking into that!

**Comment by daichety** [comment]
hello there, 
with the latest update to version 1.28.0 the issue with the actions was solved. 
Unfortunately cohorts still don't populate from either actions or events.
It's loading endlessly instead. 

<img width="1312" alt="Screenshot 2021-09-20 at 22 04 58" src="https://user-images.githubusercontent.com/63511071/134067825-dbb1cac6-c211-4571-9e50-3bbe35598a2a.png">

Furthermore, past cohorts don't update although there are new persons who would fall in the given criteria (see screenshot of a cohort I created 5 months ago - the last calculation is from 3 months ago). 
 
Thanks!
